### ISSUE TITLE
Error when trying conic lower level with Ipopt

### ISSUE THREAD

**Issue by alejandroowen** [joaquimg/BilevelJuMP.jl#96]
I tried the following code I found in the readme:

```
IPO_OPT = Ipopt.Optimizer(print_level=0)
IPO = MOI.Bridges.Constraint.SOCtoNonConvexQuad{Float64}(IPO_OPT)
BilevelModel(IPO, mode = BilevelJuMP.ProductMode(1e-5))
```

And I get the following error:

![image](https://user-images.githubusercontent.com/69917992/107582636-c3a58280-6bc7-11eb-8ff6-5d868fba0e5d.png)

**Comment by joaquimg** [comment]
You can do:
```julia
using BilevelJuMP, Ipopt, JuMP
IPO_OPT = Ipopt.Optimizer(print_level=0)
IPO = MOI.Bridges.Constraint.SOCtoNonConvexQuad{Float64}(IPO_OPT)
BilevelModel(()->IPO, mode = BilevelJuMP.ProductMode(1e-5))
```
I replaced `IPO` by `()->IPO` as the message suggested and it worked.

**Comment by joaquimg** [comment]
I fixed the readme on the PR #97.
Thanks for pointing it out!
